packages/doc/content/components/components/bottomsheet/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: 'BottomSheet'
+metaTitle: 'BottomSheet'
+metaDescription: 'BottomSheet Component'
+---
+
+import BottomSheetWeb from './bottomsheet-web.mdx';
+
+# BottomSheet
+
+The component Bottom sheet notification is a variation of the Bottom sheet component and has the objective to provide maximum priority feedback for the user.
+
+
+ <BottomSheetWeb />
+
+  
```

Comment:
How about taking out the space on line 15 and 16?

Reply:
✅